From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Sat, 5 Oct 2019 14:47:21 +0200
Subject: [PATCH] Revert "Linux 5.3.2"
References: bnc#1151927
Patch-mainline: 5.3.3
Git-commit: 212a39fa002a3ef49a0dcd915bc1267cd8c8eb72

This reverts commit 9c30694424ee15cc30a23f92a913d5322b9e5bd2

That wasn't a 5.3.2 release, it was a 5.3.3 release, as the Makefile
changed, but the text in the commit message didn't.

So revert it, so we can do an "empty" 5.3.3 release, just to keep things
working properly.

Sorry for the noise :(

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a5f4e184b552..13fa3a409ddd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 3
-SUBLEVEL = 3
+SUBLEVEL = 2
 EXTRAVERSION =
 NAME = Bobtail Squid
 
-- 
2.23.0

